cypress/tests/us/home.cy.js

```diff
@@ -0,0 +1,32 @@
+import { checkCountryRedirect } from "../../utils/checkCountryRedirect";
+
+describe("PolicyEngine Homepage", () => {
+  const regexArg = /the\s(us|u\.s\.|usa|united states)/;
+
+  beforeEach(() => {
```

Comment:
When I run these tests locally, none of the tests actually run, as I get this error here:
```
  Running:  home.cy.js                                                                      (1 of 1)


  PolicyEngine Homepage
    1) "before each" hook for "should redirect from root to US homepage by default"


  0 passing (747ms)
  1 failing

  1) PolicyEngine Homepage
       "before each" hook for "should redirect from root to US homepage by default":
     CypressError: `cy.visit()` failed trying to load:

/

We failed looking for this file at the path:

/Users/administrator/Documents/forks/scottrosen14/policyengine-app/

The internal Cypress web server responded with:

  > 404: Not Found
```

I admit, it may be my setup, as I'm unfamiliar with Cypress, but wanted to highlight this.